signalflow/computation.go

```diff
@@ -183,7 +201,13 @@ func (c *Computation) processMessage(m messages.Message) {
 			c.maxDelayMS = pointer.Int(v.MessageBlock.Contents.(messages.JobInitialMaxDelayContents).MaxDelayMS())
 		}
 	case *messages.ErrorMessage:
-		c.lastError = fmt.Errorf("error executing SignalFlow: %v", v.RawData())
+		rawData := v.RawData()
+		computationError := ComputationError{
+			Code:      rawData["error"],
```

Comment:
I have confirmed by looking at the backend code that `error` will always be an `int`, `message` will always be a `string`, and `errorType` will always be a `string`. I think it is pretty safe using those types even though it isn't officially documented. Would also make it easier to inspect without having to do a bunch of type assertions/switches in Go.